### ISSUE TITLE
Colors no longer obeyed in 1.4.0

### ISSUE THREAD

**Issue by DannyBen** [sorenisanerd/gotty#41]
It seems that the color preferences are no longer respected in 1.4.0.
I was looking through the release notes, and did not see anything to explain it.

### What version of GoTTY are you using (`gotty --version`)?

Gotty 1.4.0 - bug exists
Gotty 1.3.0 - bug does not exist

### What operating system and browser are you using?

Ubuntu 20.04

### What did you do?

Have this file in `~/.gotty`

```
port = 3000

term = "hterm" 

preferences {
  background_color = "white"
  foreground_color = "black"
  cursor_color = "blue"
}
```

and run `gotty --once ls`

Note: This exact same config worked in all previous Gotty versions, including 1.3.0

### What did you expect to see?

The terminal background color should be white.

### What did you see instead?

The terminal background color was black.

**Comment by DannyBen** [comment]
I see now that [hterm was removed](https://github.com/sorenisanerd/gotty/commit/163fd05) - does this mean that there is no longer color customization?

**Comment by sorenisanerd** [comment]
Yes, unfortunately, that's the current state of affairs. I'm going to try to add some options in the UI to change colors, fonts, etc.

**Comment by DannyBen** [comment]
> Yes, unfortunately, that's the current state of affairs. I'm going to try to add some options in the UI to change colors, fonts, etc.

I was afraid of that.... that is a deal breaker for me. Thanks for trying.

**Comment by sorenisanerd** [comment]
I'll also add an option to set the default theme once I get that far. That should let you do what you want.

**Comment by sorenisanerd** [comment]
This should (finally!) be fixed now. See https://github.com/sorenisanerd/gotty#display-customization for info.

If that doesn't meet your needs, feel free to re-open and let me know what I missed :)

**Comment by DannyBen** [comment]
Thanks. I am using it less now, but still interested and happy to know it is still maintained. I might try it again.
